.gitlab-ci.yml

```diff
@@ -180,6 +180,23 @@ windows:
     paths:
     - parity.zip
     name: "x86_64-pc-windows-msvc_parity"
+android-armv7:
+  stage: build
+  image: parity/parity-android:latest
+  only:
+    - beta
+    - tags
+    - stable
+    - triggers
+  script:
+    - cargo build --target=armv7-linux-androideabi
```

Comment:
Hey, @tomaka, does this even make sense? This just builds it. Wouldn't it be nice to have the artifacts published via vanity/releases?

Reply:
Shouldn't we try to create a complete build job like this?

```
     - scripts/gitlab-build.sh armv7-linux-androideabi armv7-linux-androideabi armhf arm-linux-androideabi-gcc arm-linux-androideabi-g++ android
```

Reply:
@5chdn In this PR, I just copy-pasted the code of ARM-Linux and didn't think much about it.
However I don't think it's useful to have artifacts. We want to make sure that Parity continues to build on Android, but I think the main usage of Parity on Android will be as a library.

Reply:
Ok perfect!